### DISCUSSION TITLE
GSoC 24 - Using AI for Unicode Spoof Detection

### DISCUSSION THREAD

**Discussion by RaidedCluster** [unicode-org/icu4x#4731] [General]
Hey everyone.

I'm Johan, a freshman studying Comp Sci & Jr. ML dev at 合同会社 107 / 107 LLC to pay off college tuition.

I found _Using AI for Unicode Spoof Detection_ a very interesting topic to work on, as Latin-look-alikes & Letter-like symbols are often used for spoofing and bypassing guardrails, especially in LLMs.

I've reported 3 relevant vulnerabilities to Khan Academy and I am also writing a paper about this with a professor.

Looking forward to share my findings and contribute to systems affected by this!

**Comment by sffc** [comment]
Thank you for reaching out. I also see the email you sent us and I replied to you there.
